src/scripts/quests/Quest.ts

```diff
@@ -25,11 +25,10 @@ abstract class Quest {
         this.notified = false;
     }
 
-    claimReward() {
+    exit() {
```

Comment:
i feel like `exit` isn't very descriptive of the function,
Do you think something like `endQuest` would work better?
```suggestion
    endQuest() {
```